doc/reference/zio/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _zio_api:
+
+.. comment
+   not documenting
+   .. doxygengroup:: display_interfaces
+
+ZIO APIs
+########
+
+ZIO Device API
+**************
+
+.. doxygengroup:: zio_device
+   :project: Zephyr
+
+ZIO Attribute Definitions
+*************************
+
+.. doxygengroup:: zio_attributes
```

Comment:
This doxygen group isn't defined (in this PR)

Reply:
doh, thanks, will fix this up

Reply:
I've removed that doxygen group for now